BalanceSheet.py

```diff
@@ -521,14 +386,23 @@ def quickRatio( cashEquivalents, aRecievable, currentLiabilities):
     Cash & Equivalents + A/R) / CurrentLiabilities
     """
 
-""" 
+""" Ratios:
 CashRatio = cash / CurrentLiabilities
 CurrentRatio = CurrentAssets / CurrentLiabilities.
-Quick Ratio = (Cash & Equivalents + A/R) / CurrentLiabilities
+Quick Ratio = (Cash & Equivalents + Accounts Receivable (A/R) / CurrentLiabilities
 
 """
 
-"We want estimated Doubtful Debt, first:"
+"""

```

Comment:
Improved this comment: `Doubtful Debt` calculation 
although abit far, yet necessary in getting things more clear:
1. adds Link to the source
2. adds an example (from source)